driver/csiplugin/connectors/rest_v2.go

```diff
@@ -1333,3 +1333,18 @@ func (s *spectrumRestV2) IsNodeComponentHealthy(nodeName string, component strin
 
 	return true, nil
 }
+
+func (s *spectrumRestV2) SetFilesystemPolicy(policy *Policy, filesystemName string) error {
+	glog.V(4).Infof("rest_v2 setFilesystemPolicy for filesystem %s", filesystemName)
+
+	setPolicyURL := utils.FormatURL(s.endpoint, fmt.Sprintf("scalemgmt/v2/filesystems/%s/policies", filesystemName))
+	status := Status{}
+
+	err := s.doHTTP(setPolicyURL, "PUT", &status, policy)
+	if err != nil {
+		glog.Errorf("Unable to set filesystem policy: %v", status.Message)
```

Comment:
Maybe we can print the rule here as well, or not sure if status.Message includes it. That way we can debug why the policy installation failed.

Reply:
Changed to check the status of the async job vs the start job request. This does return the stderr of the command if there was an error which will print things like sql syntax errors, etc.